tkltest/util/unit/build_util.py

```diff
@@ -38,7 +38,7 @@ def __get_jars_for_tests_execution():
         'junit-4.13.1.jar',
```

Comment:
I think you can use the other `required_lib_jars` variable defined at the top of this file